taskcat/cfn/template.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, template_path: str, project_root: str = '', url: str = '',
         self.template = cfnlint.decode.cfn_yaml.load(self.template_path)
         with open(template_path, 'r') as fh:
             self.raw_template = fh.read()
+        with open(template_path, 'r') as fh:
```

Comment:
A bit concerned that we're fetching and storing the same (potentially sizable) data in 2 places. Could we rather add a method that utilizes the existing raw_template, something like

```
    def template_split_by_line(self):
        return self.raw_template.split('\n')
```

or vice-versa and store the line-split version, and have a method to get it as one big string.